encoding/thrift/thriftrw-plugin-yarpc/golden_test.go

```diff
@@ -210,7 +216,12 @@ nc %v %v
 }
 
 func thriftrw(args ...string) error {
-	cmd := exec.Command("thriftrw", args...)
+	root, err := filepath.Abs("../../..")
+	if err != nil {
+		return fmt.Errorf("failed to resolve absolute path to project root: %v", err)
+	}
+
+	cmd := exec.Command("go", append([]string{"run", "-mod=vendor", root + "/vendor/go.uber.org/thriftrw"}, args...)...)
```

Comment:
Run thriftrw based on go.mod version (therefore `go mod vendor needed`), not against local installation.